src/command/mod.rs

```diff
@@ -35,6 +35,10 @@ pub enum Command {
     /// 🎆  pack up your npm package and publish!
     Publish {
         /// The path to the Rust crate.
+        #[structopt(long = "access", short = "a")]
+        access: Option<Access>,
+
+        /// The access level for the package to be published
```

Comment:
This comment should be on the field above, and the comment for the field above should be here :-p

Reply:
d'oh. good catch.

Reply:
This is still unaddressed ;)